src/core/mapper.rs

```diff
@@ -14,6 +16,8 @@ pub struct Virtual {
     disks: Vec<DiskPath>,
     vdisks: VDisksMap,
     nodes: NodesMap,
+    disk_access_sems: Arc<std::sync::RwLock<SemMap>>,
```

Comment:
Why do you use std RwLock instead of async?

Reply:
Used this one for tests. Changed to async rwlock